packages/flutter/lib/src/painting/continuous_stadium_border.dart

```diff
@@ -0,0 +1,278 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'basic_types.dart';
+import 'borders.dart';
+import 'edge_insets.dart';
+
+/// Creates a continuous stadium shape.
```

Comment:
Same here. Since this class doesn't perform an action, let the description be a noun.